pubsublite/streaming-analytics/README.md

```diff
@@ -0,0 +1,107 @@
+# Pub/Sub Lite with Cloud Dataflow
+
+[![Open in Cloud Shell](http://gstatic.com/cloudssh/images/open-btn.svg)](https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/GoogleCloudPlatform/java-docs-samples&page=editor&open_in_editor=pubsublite/streaming-analytics/README.md)
+
+Samples showing how to use [Pub/Sub Lite] with [Cloud Dataflow].
+
+## Pub/Sub Lite to Cloud Storage sample
+
+[PubsubliteToGcs.java](examples/PubsubliteToGcs.java)
+
+This sample shows how to create an [Apache Beam] streaming pipeline that reads
+messages from [Pub/Sub Lite], group the messages using a fixed-sized windowing
+function, and writes them to [Cloud Storage].
+
+Resources needed for this example:
+
+1. A pair of Pub/Sub Lite topic and subscription. 
+2. A Cloud Storage bucket.
+
+### Setting up
+
+1. [Enable the APIs](https://console.cloud.google.com/flows/enableapi?apiid=dataflow,compute_component,logging,storage_api,pubsublite.googleapis.com): Cloud Dataflow, Compute Engine, Cloud Logging, Cloud Storage, Pub/Sub Lite.
+
+1. Create a Cloud Storage bucket.
+
+   ```bash
+   PROJECT=$(gcloud config get-value project)
+   BUCKET=your-gcs-bucket
+   
+   gsutil mb gs://$BUCKET
+   ```
+   
+ 1. Create a Pub/Sub Lite topic and subscription. Set `LITE_LOCATION` to a [Pub/Sub Lite location].
+ 
+    ```bash
+    TOPIC=your-lite-topic
+    SUBSCRIPTION=your-lite-subscription
+    LITE_LOCATION=your-lite-location
+    
+    gcloud pubsub lite-topics create $TOPIC --zone=$LITE_LOCATION --partitions=1 \
+        --per-partition-bytes=30GiB
+    gcloud pubsub lite-subscriptions create $SUBSCRIPTION --zone=$LITE_LOCATION --topic=$TOPIC
+    ```
+
+1. Set `DATAFLOW_REGION` to a [Dataflow region] close to your Pub/Sub Lite location.
+
+   ```
+   DATAFLOW_REGION=your-dateflow-region
+   ```
+   
+### Running the example
+
+[PubsubliteToGcs.java](examples/PubsubliteToGcs.java)
+
+The following example runs a streaming pipeline. Choose `DirectRunner` to test it locally or `DataflowRunner` to run it on Dataflow.
+
++ `--subscription`: the Pub/Sub Lite subscription to read messages from
++ `--output`: the full filepath of the output files
++ `--windowSize [optional]`: the window size in minutes, defaults to 1
++ `--runner [optional]`: `DataflowRunner` or `DirectRunner`
++ `--project [optional]`: your project ID, optional if using `DirectRunner`
++ `--region [optional]`: the Dataflow region, optional if using `DirectRunner`
++ `--tempLocation`: the temporary location of Dataflow files, optional if using `DirectRunner`
```

Comment:
Can we reword to something along these lines?

a Cloud Storage location for temporary files if using `DataflowRunner`, optional if using `DirectRunner`